test/vm-prep

```diff
@@ -79,6 +79,12 @@ EOF
         echo "$rule" >> ${qemu_config_dir}/bridge.conf
       fi
   done
+
+    # We really want nested virtualization
+    if [ ! -f /etc/modprobe.d/kvm-intel.conf ]; then
```

Comment:
This assumes that all test runners have `kvm_intel` and not `kvm_amd`. Also, `vm-prep` is intended to prepare all tests, but the verify ones are only a subset. Should we really impose these two constraints for all test runners?